content/02.introduction.md

```diff
@@ -24,6 +24,74 @@ We designed a series of experiments to determine the extent to which label funct
 We examine relationships that indicate similar types of physical interactions (i.e., gene-binds-gene and compound-binds-gene) as well as different types (i.e., disease-associates-gene and compound-treats-disease).
 The re-use of label functions could dramatically reduce the number required to generate and update a heterogeneous knowledge graph.
 
-## Recent Work
+### Related Work
 
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+Relationship extraction is the process of detecting and classifying semantic relationships from a collection of text.
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and identifying key text patterns for relationship extraction, (2) the use of unsupervised methods via co-occurrence scores or clustering, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+In this section, we discuss selected efforts for each type of edge that we include in this project.
+
+#### Disease-Gene Associations 
+
+Efforts to extract Disease-associates-Gene (DaG) relationships have often used manually crafted rules or unsupervised methods.
+One study used hand crafted rules based on a sentence's grammatical structure, represented as dependency trees, to extract DaG relationships [@tag:pkde4j].
+Some of these rules inspired certain DaG text pattern label functions in our work.
+Another study used co-occurrence frequencies within abstracts and sentences to score the likelihood of association between disease and gene pairs [@tag:diseases].
+The results of this study were incorporated into Hetionet v1, so this served as one of our distant supervision label functions.
+Another approach built off of the above work by incorporating a supervised classifier, trained via distant supervision, into their scoring scheme [@tag:cocoscore].
+Each sentence containing a disease and gene mention is scored using a logistic regression model.
+These scores are combined using the same co-occurrence approach used in [@tag:diseases].
+We compared our results to this to measure how well our overall approach performs relative to other methods.
+Besides the mentioned three studies, other work have used co-occurrence for extraction [@tag:lgscore; @tag:polysearch; @tag:full_text_co_abstracts] or in combination with other features to mine the DaG relationships [@tag:dg_text_pubmed].
+Along with co-occurrence, one study used a bi-clustering approach to detect DaG relevant sentences from Pubmed abstracts [@tag:global_network].
+This clustering approach used dependency paths to group similar sentences together.
+We used the results of this work for our domain heuristic label functions.
+The benefit of using the above approaches is inessential need for annotated data.
```

Comment:
```suggestion
These approaches do not rely on a well-annotated training performance and tend to provide excellent recall, though the precision is often worse than with supervised methods.
```
this assertion needs a reference and/or a set of references if you're synthesizing across what you've read in multiple papers.